src/library/solrize.py

```diff
@@ -96,6 +96,48 @@ def process_hash_list(document_datasets):
         doc.solr_api_error
     """
 
+    def delete_from_solr(cores: dict, file_id: str, file_hash: str, query: dict):
```

Comment:
Is there a reason this and the following one are defined in process_hash_list and not just at the top level?

Reply:
I put them here since there are only used here at the moment.

Reply:
Hmm .... I think you could say that about a lot of code in refresher, and it's all at top level. And maybe we'll write tests one day (tho these funcs are probably lower on the list of priority candidates). This way left me wondering if there was a special reason for it. I think I'd prefer it at top level but that's not a strongly held view. 

(Also, will it be faster to have them defined outside a loop so Python only defines them once? Tho probably very little difference there compared to other speedups we could be working on)

Reply:
No special reason, just wanted to keep top level clean as possible. But makes sense to move it if we might refactor further and make tests at some point.